release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -74,10 +74,13 @@ With this release, {product-title} {product-version} introduces a {op-system-bas
 ==== VMware vSphere region and zone enablement
 You can deploy an {product-title} cluster to multiple vSphere datacenters or regions that run in a single VMware vCenter. Each datacenter can run multiple clusters or zones. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
 
-You can also specify multiple regions and zones on a cluster after installation. You must upgrade your existing cluster to {product-title} {product-version} before you configure a multiple region and zone environment.
+[IMPORTANT]
+====
+For {product-title} {product-version}, you cannot deploy an upgraded cluster to multiple vSphere regions in a VMware vCenter instance. You must install a cluster on {product-title} {product-version}, configure the multiple vSphere datacenter environment, and then deploy the cluster in this environment. 
```

Comment:
Instead of "deploy an upgraded cluster" I'd explain something along these lines:

"You can only use this feature in newly installed clusters. This feature requires the CSI storage class which is only available in newly installed clusters"

Reply:
Updated